ZenPacks/zenoss/RabbitMQ/modeler/plugins/zenoss/ssh/RabbitMQ.py

```diff
@@ -136,18 +138,33 @@ def getExchangeRelMap(self, exchanges_string, compname):
             if not exchange_string.strip():
                 continue
 
-            name, exchange_type, durable, auto_delete, arguments = \
-                re.split(r'\s+', exchange_string)
-
-            if not name:
-                name = 'amq.default'
+            federated = False
+            exchange_string = exchange_string.strip()
+            if exchange_string.startswith('federation'):
+                federated = True
+                split_exchange = re.split(r'\s+', exchange_string.strip())
+                if len(split_exchange) == 9:
```

Comment:
I don't think they are in this case- unless I missed it.  But yeah, if you're intentionally ignoring that row, i'd log a warning and continue